### ISSUE TITLE
CustomView with ListView doesn't scroll

### ISSUE THREAD

**Issue by david145** [afollestad/material-dialogs#766]
Hey, 

I've added a custom view to my dialog, basically it looks like this:

```xml
    <LinearLayout>

     <ListView/>

    <LinearLayout>

        <TextView/>
        <TextView/>

    </LinearLayout>

    <LinearLayout>

        <TextView/>
        <TextView/>

    </LinearLayout>

    <LinearLayout>

        <TextView/>
        <TextView/>

    </LinearLayout>

    <View/>

    <LinearLayout>

        <TextView/>
        <TextView/>

    </LinearLayout>

    <LinearLayout>

        <TextView/>
        <TextView/>

    </LinearLayout>

    </LinearLayout>
```

Problem is, I can't scroll. I've set

```java
       .customView(R.layout.xyz, false)
```

When set in on true to wrap it in a scroll view it's working great and I can scroll, but I get some paddings, which I don't want. Does anyone has an idea why I can't scroll with my first mentioned solution or tell me how  I can wrap it in a scroll view like the customView function does, just without getting the paddings?

Thanks!

**Comment by afollestad** [comment]
Wrapping in a ScrollView literally means wrapping your content in a ScrollView 😜

```java
<ScrollView>

<LinearLayout>

// Children here

</LinearLayout>

</ScrollView>
```

**Comment by afollestad** [comment]
Oh, but there's a ListView in there too. It'll scroll once the dialog is at max height, in my experience.

**Comment by david145** [comment]
Yeah, that's the point, if I literally wrap my Layout in a Scrollview I get an error. But with your wrapInScrollView variable it works?

![screenshot_2015-10-18-12-42-04](https://cloud.githubusercontent.com/assets/11370593/10562110/2a2c3234-7596-11e5-9707-aac070ca3be3.png)

It seems the max height is reached, but I just can't scroll.

**Comment by afollestad** [comment]
Hmmm, that's very odd. I'll look into it.

**Comment by Jawnnypoo** [comment]
Maybe try it with a `NestedScrollView` instead of a `ScrollView`?

**Comment by enricocid** [comment]
Try to add this to ListView:

android:layout_height="0dp"
android:layout_weight="1"
